examples/tf_model_parallelism/example.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
```

Comment:
Let's get rid of this file. We can include it in `examples/` when we're ready to advertise the feature.